app/views/ctc/questions/spouse_filed_prior_tax_year/_pr_edit.html.erb

```diff
@@ -0,0 +1,22 @@
+<% @main_question = t("views.ctc.questions.spouse_filed_prior_tax_year.puerto_rico.title", spouse_first_name: current_intake.spouse_first_name, prior_tax_year: prior_tax_year) %>
```

Comment:
@embarnard and I changed a different `_pr_edit.html.erb` to `_puerto_rico_edit.html.erb` a day or two ago. I like the clarity of spelling out `puerto_rico`, so I encourage it here. It doesn't matter a lot, and Em and I haven't done something like presented at Computer Gardening Club about it yet, so not a blocking comment. I do think it'd be nice to be consistent with the spelled-out `_puerto_rico_edit.html.erb`.